### PR TITLE
[templates] merge nearlib and not nearlib templates

### PR DESCRIPTION
**Pull request by janedegtiareva**

### CODE REVIEW

File: templates/todomvc/src/main.js [hunk 1]
```diff
@@ -1,11 +1,77 @@
...
+  near.require = async function(contractName, viewMethods, changeMethods) {
+    let nearConfig = await this.getConfig();
+
+    // TODO: Introspection of contract methods
+    let contract = {};
+    viewMethods.forEach((methodName) => {
+      contract[methodName] = async function(args) {
+        //console.log(methodName, args);
+        args = args || {};
+        let response = await sendJson("POST", `${nearConfig.baseUrl}/contract/view/${contractName}/${methodName}`, { args });
```
Comment (vgrichina) [inline templates/todomvc/src/main.js]: this still goes around nearlib

File: templates/todomvc/src/main.js [hunk 2]
```diff
@@ -1,11 +1,77 @@
+let near = null;
+
+async function init() {
```
Comment (vgrichina) [inline templates/todomvc/src/main.js]: boilerplate code needs to live in Nearlib, not in `main.js`

Comment (janedegtiareva) [reply templates/todomvc/src/main.js]: I am not a big fan of putting this in nearlib. How about some other file specific to nearstudio?

File: templates/todomvc/src/main.js (hunk 1 above)
Comment (janedegtiareva) [reply templates/todomvc/src/main.js]: Not sure what you mean by this

Comment (vgrichina) [reply templates/todomvc/src/main.js]: I mean that this code calls contract helper. It should be calling nearlib instead.

File: templates/todomvc/src/main.js (hunk 2 above)
Comment (vgrichina) [reply templates/todomvc/src/main.js]: imo, this should just go to browser nearlib only (i.e. `exports.js`).

**Pull request closed without merging**